### ISSUE TITLE
[css-contain-3] container-type syntax implies you might need multiple values

### ISSUE THREAD

**Issue by mirisuzanne** [w3c/csswg-drafts#7977]
Currently, the [formal syntax for `container-type`](https://www.w3.org/TR/css-contain-3/#container-type) is:

```
normal || [ size | inline-size ]
```

This implies you might want to use `normal` _along side_ either `size` or `inline-size`. The only reason to do that would be if `inline-size` acted as an override to the `normal` behavior - in which case authors would need to re-assert `normal inline-size` as a way to query both styles and sizes of the container. But as far as I can tell, that's not the behavior described in the spec, or the desired behavior, or the way things work with the Chrome prototype of style queries. Instead, a container type of `inline-size normal` [behaves the same](https://codepen.io/miriamsuzanne/pen/poKJxLO) as a container type of `inline-size`.

I think we should probably change the syntax to: 

```
normal | size | inline-size
```

But, alternately, we could just add a note to clarify how this works.

**Comment by Loirooriol** [comment]
Mm, already fixed in #7669? https://drafts.csswg.org/css-contain-3/#container-type

**Comment by mirisuzanne** [comment]
ha, you're right. thaks
